viewer/src/views/threejs/OrderSectorsByVisibilityCoverage.ts

```diff
@@ -291,7 +291,7 @@ export class GpuOrderSectorsByVisibilityCoverage {
       addSector(sector.bounds, sector.id, coverageFactors);
     });
 
-    const buffer = new THREE.InstancedInterleavedBuffer(instanceValues, 5);
+    const buffer = new THREE.InstancedInterleavedBuffer(instanceValues, 4);
```

Comment:
Why has this been changed? Was it the wrong number in the beginning or was the data format changed?